modules/mayaUSD_Win.mod.template

```diff
@@ -5,6 +5,7 @@ presets:
 scripts: lib/python
 resources: 
 PYTHONPATH+:=lib/python
+${MAYAUSD_EXTRA_PYTHON_PATH}
```

Comment:
Sorry, should have remembered sooner... Needs to be the last line of that block. The `.mod` parser interprets a blank line as the end of the section.